Arriba/Arriba.Web/parts/Index.jsx

```diff
@@ -0,0 +1,16 @@
+import Search from "./Search";
+import Grid from "./Grid";
+window.configuration = require("../configuration/Configuration.jsx").default;
+
+class Index extends React.Component {
+    render() {
+        return window.location.pathname.startsWith("/Grid.html")
+            ? <Grid params={getQueryStringParameters()}
+                url={configuration.url} 
```

Comment:
Should we also stop passing configuration props to Grid, and have it access configuration.X directly?